modules/javafx.web/src/main/native/Source/WebKitLegacy/java/WebCoreSupport/HistoryItemClientJava.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
```

Comment:
Presuming we need this file, it will need a proper copyright header.

Reply:
Updated the copyright